packages/aws-cdk-lib/aws-s3/test/bucket.test.ts

```diff
@@ -3463,6 +3464,26 @@ describe('bucket', () => {
     Template.fromStack(stack).resourceCountIs('AWS::Lambda::Function', 1);
   });
 
+  test('with autoDeleteObjectsLogGroup', () => {
+    const stack = new cdk.Stack();
+
+    new s3.Bucket(stack, 'MyBucket', {
+      removalPolicy: cdk.RemovalPolicy.DESTROY,
+      autoDeleteObjects: true,
+      autoDeleteObjectsLogGroup: new logs.LogGroup(stack, 'LogGroup1', {
+        logGroupName: 'MyLogGroup',
+      }),
+    });
+
+    Template.fromStack(stack).hasResourceProperties('AWS::Lambda::Function', {
+      LoggingConfig: {
+        LogGroup: {
+          'Ref': 'LogGroup106AAD846',
+        },
+      },
+    });
+  });
```

Comment:
nit: should we also be checking if the log group resource was created?

Reply:
Good idea. Will update